proxy/src/http/project.rs

```diff
@@ -505,4 +532,26 @@ mod test {
 
         Ok(())
     }
+
+    #[tokio::test]
+    async fn discover() -> Result<(), error::Error> {
+        let tmp_dir = tempfile::tempdir()?;
+        let ctx = http::Context::tmp(&tmp_dir).await?;
+        let api = super::filters(ctx.clone());
+
+        let ctx = ctx.read().await;
+        let key = ctx.keystore.get_librad_key()?;
+        let owner = ctx.peer_api.init_owner(key.clone(), "cloudhead")?;
+
+        coco::control::setup_fixtures(&ctx.peer_api, key, &owner)?;
+
+        let feed = project::discover()?;
+        let res = request().method("GET").path("/discover").reply(&api).await;
+
+        http::test::assert_response(&res, StatusCode::OK, |have| {
+            assert_eq!(have, json!(feed));
```

Comment:
It would be nice to assert not only the self-referencing test here but also that we expect a populated json response.